packages/user-operation-controller/CHANGELOG.md

```diff
@@ -6,4 +6,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-[Unreleased]: https://github.com/MetaMask/core/
+## [1.0.0]
```

Comment:
Hmm, since this is the initial release of this controller, it would be particularly easy to postpone it.

cc @matthewwalsh0, do you think this controller is ready for release?

Reply:
I removed the user operations controller from this release for now